spyder/plugins/ipythonconsole/widgets/shell.py

```diff
@@ -205,8 +175,7 @@ def __init__(self, ipyclient, additional_options, interpreter_versions,
         self.ipyclient = ipyclient
         self.additional_options = additional_options
         self.interpreter_versions = interpreter_versions
-        self.is_external_kernel = is_external_kernel
-        self.is_spyder_kernel = is_spyder_kernel
+        self.kernel_handler = False
```

Comment:
```suggestion
        self.kernel_handler = None
```

I think this should `None` instead of `False`, right?